ui/admin/app/styles/app.scss

```diff
@@ -951,3 +951,37 @@
     margin-bottom: 1.5rem;
   }
 }
+
+// storage policies
+.policy {
+  margin-bottom: 0;
+}
+
+.policy-custom-input {
+  .hds-form-field__control {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    .hds-segmented-group {
+      width: 45%;
+    }
+
+    .hds-form-field--layout-flag {
+      width: 40%;
+      margin-bottom: 0;
+    }
+  }
+
+  .hds-badge {
```

Comment:
Could you have used just a normal segmented group select with only one option but made it `readonly` so the user can't change it rather than using a badge and restyling it?

Reply:
Hmm, In that case, I would still need to update the style to remove the arrows on the input

Reply:
resolving this to get this PR merged. will address it later if it's still a concern